Workaround to build with SDL 2.24.1+ until upstream catches up to new
SDL versioning

Index: libraries/textscreen/txt_sdl.h
--- libraries/textscreen/txt_sdl.h.orig
+++ libraries/textscreen/txt_sdl.h
@@ -23,7 +23,7 @@
 
 #include "SDL.h"
 
-#if (SDL_MAJOR_VERSION == 2 && SDL_MINOR_VERSION == 0)
+#if (SDL_MAJOR_VERSION == 2)
 	#define SDL20
 #elif (SDL_MAJOR_VERSION == 1 && SDL_MINOR_VERSION == 2)
 	#define SDL12
